src/index.js

```diff
@@ -14,20 +14,22 @@ governing permissions and limitations under the License.
 const Swagger = require('swagger-client')
 const { codes } = require('./SDKErrors')
 const logger = require('@adobe/aio-lib-core-logging')('aio-lib-analytics', { level: process.env.LOG_LEVEL })
+const fetch = require('node-fetch')
```

Comment:
I think you forgot to `npm install node-fetch`. If you did, you probably forgot to add the updated `package.json` in this PR

Reply:
You are right.  I've resolved that in the latest PR.